crates/exex/exex/src/manager.rs

```diff
@@ -377,6 +383,14 @@ where
                 .unwrap();
 
             self.wal.finalize(lowest_finished_height)?;
+            if self.wal.num_blocks() > (DEFAULT_MAX_WAL_BLOCKS_WARNING as u64) {
+                warn!(
+                    target: "exex::manager",
+                    "WAL contains {} blocks, exceeding the limit of {}",
+                    self.wal.num_blocks(),
+                    DEFAULT_MAX_WAL_BLOCKS_WARNING,
```

Comment:
I would rephrase this to something that asks the user to check that their ExEx is behaving correctly and emits `FinishedHeight` event, because technically there's no limit for the WAL size, and it's just a pure "FYI" log.

Reply:
Thanks @shekhirin for your reviewing. Can you suggest me a warning message. I will fix it in the next commit.